packages/millicast-sdk/src/Publish.js

```diff
@@ -257,4 +282,19 @@ export default class Publish extends BaseWebRTC {
       })
     }
   }
+
+  /**
+   * Send SEI user unregistered data as part of the frame being streamed. Only available for H.264 codec.
+   * @param {String} message String with the data to be sent as SEI user unregistered data.
+   * @param {String} [uuid="6e9cfd2a-5907-49ff-b363-8978a6e8340e"] String with UUID format as hex digit (XXXX-XX-XX-XXXXXX).
+   */
+  sendMetadata (message, uuid = DOLBY_SEI_DATA_UUID) {
```

Comment:
Should we block non H.26x stuff here?

Reply:
We have better visualization on codec status once entered the transform worker, from the `encodedFrame.getMetadata()` function, but if it is necessary or better to do it here we can look for other options, maybe looking over the SDP or a new global variable.

Reply:
Sorry, missed it - if it's being done then no worries at all